client/scripts/controllers/server/tags.ts

```diff
@@ -103,14 +104,13 @@ class TagsController {
     }
   }
 
-  public async refreshAll(communityId?, chainId?, reset = false) {
+  public async refreshAll(chainId, communityId, reset = false) {
```

Comment:
I feel like these should be optional and when invoked, you just specify `.refreshAll(chainId: this.meta.id, reset: true)` or `.refreshAll(communityId: this.meta.id, reset: true)`. Forcing the function to take at least 1 null is a little unclear.

Reply:
also the rest of the logic, if passing `chainId = null`, would result in `chain: app.activeChainId()` rather than `null`. this could lead to unexpected behavior, though not experiencing anything at the moment.

Reply:
I get that, but this is the pattern used for all forum content, and the passing null is standardized right now, so it seems clearer to work with the standard than be inconsistent. It might be worth opening a ticket to change that site-wide?